### DISCUSSION TITLE
Open a channel/DM in RHS

### DISCUSSION THREAD

**Discussion by abhijit-singh** [mattermost/community#70] [Practical Ideas]
It'll be awesome to be able to have two channels open simultaneously. One way of doing this is having an `Open in RHS` button for a channel/DM from the channel menu in the LHS. 

The use case is that I want to monitor a particular channel (because I'm having a discussion / following an ongoing conversation / there's a critical incident running I want to stay updated on) while also being able to switch through other channels to check unreads and reply to messages.

**Comment by Willyfrog** [comment]
Wouldn't channel notifications set to `notify on all messages` cover this case?

I still think it would be cool to have multiple channels in a view (not only 2, but even 3)

**Comment by chetanyakan** [comment]
Threads can also be used as they can already be opened in RHS.

**Comment by esethna** [comment]
I think pop-out chats could be a solution here too: https://github.com/mattermost/community/discussions/17#discussioncomment-907208
